src/layer.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+
+class Layer:
+    def __init__(self, input_count, neuron_count, activation_function, weights = None, biases = None):
+        self.input_count = input_count
+        self.neuron_count = neuron_count
+        self.activation_function = activation_function
+        self.weights = weights if weights is not None else np.random.rand(input_count, neuron_count)
+        self.biases = biases if biases is not None else np.random.rand(neuron_count)
+
+    def activate(self, input):
+        result = np.dot(input, self.weights) + self.biases
+        output = self.activation_function(result)
```

Comment:
Zastanawiam się, czy jest sens rozbijać to na 3 linijki, czy nie lepiej  napisać:
```
return self.activation_function(np.dot(input, self.weights) + self.biases)
```